src/coreclr/jit/emitarm64.cpp

```diff
@@ -1089,6 +1089,14 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(isValidUimm4From1(emitGetInsSC(id)));
             break;
 
+        case IF_SVE_CI_3A: // ........xx..MMMM .......NNNN.DDDD -- SVE permute predicate elements
+            elemsize = id->idOpSize();
```

Comment:
Doesn't have to be done for this PR, but we should come back and clean up all these unused `elemsize` locals (and maybe update the tool to not emit them @kunalspathak).